### PR TITLE
Update jest, parcel for dependabot notice

### PR DESCRIPTION
**Pull request by palemtnrider**
Closes:  2 dependabot issues

**What changed?**
parcel and jest in package.json and package-lock.json

**Why?**
GitHub reported vulnerabilities with ansi-html, and set-value which are used by the older versions of these packages.

**How did you test it?**
I ran the UI manually against my GitOps cluster and ran the UI tests locally.

**Release notes**
N/A

**Documentation Changes**
N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,7 +72,8 @@
     },
     "modulePathIgnorePatterns": [
       "<rootDir>/dist/"
-    ]
+    ],
+    "testEnvironment": "jsdom"
```
Comment (josecordaz) [inline package.json:76]: missing coma ?

Comment (josecordaz) [reply package.json:76]: NVM this is correct

**Review by josecordaz — APPROVED** [review]
LGTM

**Comment by josecordaz** [comment]
@palemtnrider I just saw you are going to need to run `npm i` so `package-lock.json` reflects the changes made in `package.json`

**Comment by palemtnrider** [comment]
> @palemtnrider I just saw you are going to need to run `npm i` so `package-lock.json` reflects the changes made in `package.json`. This will help to pass the lint issue in CI.

Thanks @josecordaz I was just looking into that 👍

File: package.json [hunk 2]
```diff
@@ -72,7 +72,8 @@
...
   },
   "resolutions": {
```
Comment (jpellizzari) [inline package.json:78]: 💡 You can also force resolutions to specific versions here. It tends to be a little less risky compared up updating major versions.

**Review by jpellizzari — APPROVED** [review]
LGTM. I was surprised there weren't more changes necessary for the `jest` upgrade. Looks like it was only minor breakages: https://jestjs.io/blog/2021/05/25/jest-27#features-coming-with-breaking-changes

Re the failing `Git State` check, if you do all of the `npm install` operations via the `Makefile` it *should* resolve that. LMK if you need some help troubleshooting.

File: package.json (hunk 2 above)
Comment (palemtnrider) [reply package.json:78]: Thanks, @jpellizzari I _think_ I have it figured out.

**Pull request merged**